frontend_lib/src/component/DropdownMenu/DropdownMenu.styl

```diff
@@ -2,6 +2,7 @@
 
 .dropdown
   color fontColor
+  height 100%
```

Comment:
this will impact every usage of dropdown in tracim
have you checked usage of the component through tracim?

Reply:
yes, I didn't found a place where this rule was a problem